platform/darwin/src/MGLOfflineStorage.h

```diff
@@ -132,6 +133,30 @@ typedef void (^MGLOfflinePackAdditionCompletionHandler)(MGLOfflinePack * _Nullab
 typedef void (^MGLOfflinePackRemovalCompletionHandler)(NSError * _Nullable error);
 
 /**
+ The type of resource that is requested.
+ */
+typedef NS_ENUM(NSUInteger, MGLResourceKind) {
+    /** Unknown type */
+    MGLResourceKindUnknown,
+    /** Style sheet JSON file */
+    MGLResourceKindStyle,
+    /** TileJSON file as specified in https://www.mapbox.com/mapbox-gl-js/style-spec/#root-sources */
```

Comment:
Use an `<a>` tag with the text "Mapbox Style Specification", so that the documentation looks cleaner in the jazzy documentation.